data/json/items/tool/med.json

```diff
@@ -35,6 +35,7 @@
     "type": "TOOL",
     "name": { "str": "anesthesia kit" },
     "description": "A kit for inducing anesthesia for surgery, containing specialized canisters with a variety of powerful hypnotic, analgesic, and stimulative drugs.  It's intended for use in specialized medical equipment, and can't be administered manually.",
+    "category": "drugs",
```

Comment:
I'm not convinced on this. In itself this is just a container, not a drug. It's contents would be drugs, it in itself wouldn't.

Reply:
Yeah, i think anesthesia deserves a rework
I placed it in drugs so it doesn't get lost in other categories. as its really just a glorified drug.